x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchRequest.java

```diff
@@ -103,7 +100,6 @@ public EqlSearchRequest(StreamInput in) throws IOException {
             this.keepAlive = in.readOptionalTimeValue();
             this.keepOnCompletion = in.readBoolean();
         }
-        isCaseSensitive = in.readBoolean();
```

Comment:
This needs to be versions for upgrade scenarios. On the 7.x branch, there should be a :
```java
if (in.getVersion().before(V.7_10_0)) {
	in.readBoolean();
}
```

In master this won't be a problem since upgrades will happen only from the latest 7.x